_posts/platform/getting-started/2000-01-01-getting-started-with-elk.md

```diff
@@ -234,5 +234,136 @@ output {
 }
 ```
 
+## Curator
+
+### Installing Curator
+
+Since logs are only relevant for a short period of time, it is current
+practice to remove logs that are too old to be relevant. This is done to
+reduce the load on the database and limit the disk usage.
+
+This is where
+[Curator](https://www.elastic.co/guide/en/elasticsearch/client/curator/5.8/index.html)
+is needed. This project is designed to let you manage your indices life cycle.
+
+Curator is written in Python. In order to install its dependencies, you
+can modify the `.buildpacks` file to also contain the Python buildpack. The
+`.buildpacks` file should have the following content:
+
+```
+https://github.com/Scalingo/buildpack-jvm-common.git
+https://github.com/Scalingo/python-buildpack.git
+https://github.com/Scalingo/logstash-buildpack.git
+```
+
+In order to instruct the Python buildpack to install Curator and its
+dependencies, create a file named `requirements.txt` at the root of your
+project:
+
+```
+PyYAML==5.3.1
+elasticsearch-curator==5.8.0
+```
+
+
+Curator is not a deamon, it is designed as a one-off process. To be able
+to run it on Scalingo you'll need to write a simple bash script that launch
+Curator regularly. This can be achieved via a simple shell script.
+Create a file named `curator.sh` with the following content:
+
+```bash
+#!/bin/bash
+set -ex
+
+while true; do
+  echo "Running Curator"
+  curator --config curator.yml log-clean.yml
+  # Run Curator every 12h
+  sleep 43200
+done
+```
+
+Finally you need to instruct Scalingo to start Curator.
+This can be achieved by adding the following line to your `Procfile`:
+
+```yaml
+curator: ./curator.sh
+```
+
+### Configuration
+
+All the hard stuff is now done.
+
+Next step is to configure Curator. First you need to configure how Curator
+connects to your database:
+
+```
+---
+client:
+  hosts:
+    - ${ELASTICSEARCH_HOST}
+  http_auth: ${ELASTICSEARCH_AUTH}
+logging:
+  loglevel: INFO
+  logfile:
+  logformat: default
+```
+
+Curator cannot use the `ELASTICSEARCH_URL` environment variable. You
+need to define two other environment variables on your app, duplicating
+`ELASTICSEARCH_URL` content.
+Hence if your `ELASTICSEARCH_URL` variable is set to
+`http://user:password@host:port`, you need to define 2 environment variables:
+
+```
+ELASTICSEARCH_HOST=host:port
+ELASTICSEARCH_AUTH=user:password
+```
+
+The last step is to configure your indices life cycle. This is based on your
+indices names.
+When you configured Logstash, we asked you to configure your index with the
+following name `sc-apps-%{+YYYY.MM.dd}`.
```

Comment:
I think this sentence is not correctly placed. We don't really understand why you remind us about it here. It is actually related to the value of `LOGS_INDICES_PREFIX` you explain a few lines later. Maybe move this sentence later?

Reply:
You didn't addressed my question here I think

Reply:
Or just remove it.